dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchFields.java

```diff
@@ -170,6 +170,7 @@
     public static final String FILE_CHECKSUM_TYPE = "fileChecksumType";
     public static final String FILE_CHECKSUM_VALUE = "fileChecksumValue";
     public static final String FILENAME_WITHOUT_EXTENSION = "fileNameWithoutExtension";
+    public static final String FILE_EXTENSION = "fileExtension";
```

Comment:
Do we need the other constant of `FILE_TYPE_SEARCHABLE` any more?

Reply:
Yes, `FILE_TYPE_SEARCHABLE` is for `File Type` facet.